interfaces/builtin/modem_manager.go

```diff
@@ -47,6 +47,10 @@ var modemManagerPermanentSlotAppArmor = []byte(`
 # For ioctl TIOCSSERIAL ASYNC_CLOSING_WAIT_NONE
 capability sys_admin,
 
+# For {mbim,qmi}-proxy
+unix (create,bind,listen,send,receive) type=stream,
+/sys/devices/**/usb** r,
```

Comment:
The denials I see in this case are:

[64480.612240] audit: type=1400 audit(1484297690.517:195): apparmor="DENIED" operation="open" profile="snap.modem-manager.modemmanager" name="/sys/devices/pci0000:00/0000:00:14.0/usb1/1-3/descriptors" pid=7303 comm="ModemManager" requested_mask="r" denied_mask="r" fsuid=0 ouid=0

Probably MM is trying to get some information about USB endpoints. However, I found that things work even if I do not allow MM to access those files. Nonetheless, I would prefer to allow this if there is no risk.